lib/ansible/parsing/vault/__init__.py

```diff
@@ -143,48 +164,41 @@ def __init__(self, password):
         self.cipher_name = None
         self.b_version = b'1.1'
 
-    # really b_data, but for compat
-    def is_encrypted(self, data):
+    @staticmethod
+    def is_encrypted(data):
```

Comment:
They could likely also go away in favor of using the module versions everywhere. They were left in for compat, but that may not be needed.